apps/fyler/src/fyler_monitor.erl

```diff
@@ -78,6 +80,7 @@ idle(_Event, _From, State) ->
 
 monitoring(timeout, #state{alarm = false, cpu_max = Max, timeout = T} = State) ->
```

Comment:
Line 81 is too long: monitoring(timeout, #state{alarm = false, cpu_max = Max, timeout = T} = State) ->.